libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -686,6 +690,24 @@ void PbftManager::loopBackFinishState_() {
   current_step_clock_initial_datetime_ = std::chrono::system_clock::now();
 }
 
+void PbftManager::broadcastVotes(bool rebroadcast) {
+  auto [round, period] = getPbftRoundAndPeriod();
+  auto net = network_.lock();
+  if (net) {
```

Comment:
readability is better if you dont put big chunk of code into if when not needed. Instead of this if, just use:
assert(net) or

if (!net) {
  return
}

Reply:
Changed